Builds/CMake/deps/Lz4.cmake

```diff
@@ -0,0 +1,96 @@
+#[===================================================================[
+   NIH dep: lz4
+#]===================================================================]
+
+add_library (lz4_lib STATIC IMPORTED GLOBAL)
+
+find_package (PkgConfig REQUIRED)
```

Comment:
PkgConfig doesn't exist on any normal windows machine (maybe on cygwin or mingw...), so I would wrap all the search-for-lz4 stuff in an `if (NOT WIN32)` check.